walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -218,46 +139,47 @@ In this task, you use the Identity Management service to protect both the client
 
 === Add Identity Management service to the client app (PWA) and create a realm for users
 
-. Navigate to the link:{mdc-url}[Mobile Developer Console, window="_blank"].
-. Click on the *showcase-app* card to display the configuration screen.
-. Click the *Mobile Services* tab.
-. Click *Bind to App* for *Identity Management* in the *Unbound Services* section and then click the *next* button.
-.. When prompted for an *Admin User Name* enter:
-+
-----
-admin
-----
-.. When prompted for an *Admin User Password* enter:
-+
-----
-admin
-----
-. Complete the wizard.
-This process also creates a new realm in the {customer-sso-name}.
+. As your admin to create new realm for your user and obtain url to be able to login to the keycloak instance
+. In new realm we going to create two new clients - one for server side and one for the client side application.
+. Open URL obtained from adminstrator
```

Comment:
This is the most problematic part. Generally eval users will not be able to finish this walkthough without intervence of the admin user. I think @pwright has really good proposal to address this problem and this use case can be used as good example when this is needed. 

Fix for that (in integrately) will be very simple - we need to get some extra realm with users that will be matching the users created on the cluster realm (the same password). This will enable developers to explore user facing keycloak without intervention of the cluster or customer admins.

CC @wei-lee

Reply:
@wtrocki  are you proposing a change to the integreatly installer? If so, we need that to happen before this walkthrough works, right?

Reply:
@pwright I'm just flagging problem in walkthough - technically people might be able to get access to admin and create realm thenselves etc. so it might not be that kind of issue but it is possible area to improve. I just suggested possible way but maybe there is another way to do it.